src/neuroconv/schemas/base_metadata_schema.json

```diff
@@ -0,0 +1,171 @@
+  "$id": "base_metafile.schema.json",
+  "title": "Base schema for the metafile",
+  "description": "Base schema for the metafile",
+  "version": "0.1.0",
+  "type": "object",
+  "required": ["NWBFile"],
+  "properties": {
+    "NWBFile": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "keywords": {
+          "description": "Terms to search over",
+          "type": "array",
+          "items": {
+            "title": "keyword",
+            "type": "string"
+          }
+        },
+        "experiment_description": {
+          "type": "string",
+          "description": "general description of the experiment"
+        },
+        "session_id": {
+          "type": "string",
+          "description": "lab-specific ID for the session"
+        },
+        "experimenter": {
+          "description": "Name of person/people who performed experiment",
+          "type": "array",
+          "items": {
+            "type": "string",
+            "title": "experimenter"
+          }
+        },
+        "identifier": {
+          "type": "string",
+          "description": "A unique text identifier for the file. If one is not provided it will be randomly assigned"
+        },
+        "institution": {
+          "type": "string",
+          "description": "Institution(s) where experiment is performed"
+        },
+        "lab": {
+          "type": "string",
+          "description": "Lab where experiment was performed"
+        },
+        "session_description": {
+          "type": "string",
+          "description": "A description of the session where this data was generated"
+        },
+        "session_start_time": {
+          "type": "string",
+          "format": "date-time",
+          "description": "The start date and time of the recording session"
+        },
+        "surgery": {
+          "type": "string",
+          "description": "Narrative description about surgery/surgeries, including date(s) and who performed surgery."
+        },
+        "pharmacology": {
+          "type": "string",
+          "description": "Description of drugs used, including how and when they were administered. Anesthesia(s), painkiller(s), etc., plus dosage, concentration, etc."
+        },
+        "protocol": {
+          "type": "string",
+          "description": "Experimental protocol, if applicable. E.g., include IACUC protocol"
+        },
+        "related_publications": {
+          "type": "array",
+          "items": {
+            "title": "related publication",
+            "type": "string"
+          }
+        },
+        "slices": {
+          "type": "string",
+          "description": "Description of slices, including information about preparation thickness, orientation, temperature and bath solution."
+        },
+        "source_script": {
+          "type": "string",
+          "description": "Script file used to create this NWB file."
```

Comment:
Does JSON allow an 'examples' sub-field? It could be really useful info to display via the GUIDE

Like, is this field supposed to be the relative path to a file? And how would it differ from the `source_script_file_name`?

Reply:
Yeah, it's not really clear from the schema